test/e2e_node/topology_manager_test.go

```diff
@@ -868,7 +868,7 @@ func runTMScopeResourceAlignmentTestSuite(ctx context.Context, f *framework.Fram
 }
 
 func runTopologyManagerNodeAlignmentSuiteTests(ctx context.Context, f *framework.Framework, sd *sriovData, reservedSystemCPUs, policy string, numaNodes, coreCount int) {
-	threadsPerCore := getSMTLevel()
+	threadsPerCore := smtLevelFromSysFS()
```

Comment:
```suggestion
	smtLevel := smtLevelFromSysFS()

```
Above will requires also update in line 914

```
numCores := smtLevel * coreCount
```
Reasoning for improved consistency, proposed to use one common variable name to return value as done in cpu_manager_test.go and cpu_manager_metrics_test.go

Reply:
thanks, addressed